packages/website/src/components/team/TeamBio.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+export interface BioEntry {
+  description: string;
+  name: string;
+  username: string;
+}
+
+export function TeamBio({
+  description,
+  name,
+  username,
+}: BioEntry): React.JSX.Element {
+  return (
+    <li>
+      <img alt="" src={`/img/team/${username}.jpg`} />
```

Comment:
Do we need to self-host these images? You can definitely just link to `https://github.com/${username}.png`

Reply:
For performance I'd rather not incur the cost of loading images cross-domain though.

Reply:
They should be cached from GH already, surely?

Reply:
Only if you've recently visited pages on GitHub that happen to include us. So if you're a new user and just perusing the site, it won't apply.

I also am not sure how this deals with caching given the variety of sites in play... but doubt that GitHub gives as good CDN/edge-level caching as a ~static deployed site would.